### ISSUE TITLE
ktlint incorrectly reports redundant toString

### ISSUE THREAD

**Issue by SUPERCILEX** [shyiko/ktlint#69]
The `super.toString()` call is incorrectly reported even though there is no other way to get the super class's string:
```kt
override fun toString(): String = "${super.toString()}, selectedValueKey=$selectedValueKey"
```

**Comment by shyiko** [comment]
Thank you for the ticket. Fixed in 0.9.1 (it should become available through Maven Central within the next ~25 minutes).

**Comment by SUPERCILEX** [comment]
@shyiko Damn that was fast! 😄 Thanks!
